pkg/controller/appdefinition/volume.go

```diff
@@ -200,7 +201,14 @@ func toPVCs(req router.Request, appInstance *v1.AppInstance) (result []kclient.O
 			pvc.Spec.VolumeName = pvName
 
 			if volumeRequest.Size == "" {
-				pvc.Spec.Resources.Requests[corev1.ResourceStorage] = *v1.DefaultSize
+				// This shouldn't happen because we set a default in the status. However
+				// if this situation does occur, we'll just use the default size for runtime
+				// and log a warning. As an example, our unit tests hit this case.
+				if appInstance.Status.Defaults.VolumeSize == nil {
+					logrus.Debugf("no default volume size found in status, using static default size of %v", v1.DefaultSize)
```

Comment:
comment says log a warning. you are logging at debug. just wanted to check which one you want to do

Reply:
Oops. Definitely debug. Will update.